App/components/Image/Image.story.tsx

```diff
@@ -16,7 +14,33 @@ const defaultImage = () => (
     )}
     width={number('width', 128)}
     height={number('height', 128)}
-    border={select('border', BORDERS, `5px ${BORDERS[2]} gray`)}
+    border={text('border', '5px solid white')}
+    // border={{
+    //   borderStyle: select(
+    //     'borderStyle',
+    //     {
+    //       solid: 'solid',
+    //       dashed: 'dashed',
+    //       dotted: 'dotted',
+    //     },
+    //     'solid'
+    //   ),
+    //   borderColor: select(
+    //     'borderColor',
+    //     {
+    //       white: 'white',
+    //       black: 'black',
+    //       dark: 'dark',
+    //       light: 'light',
+    //       lightGray: 'lightGray',
+    //       primary: 'primary',
+    //       lightGrayBorder: 'lightGrayBorder',
+    //       grayBorder: 'grayBorder',
+    //     },
+    //     'grayBorder'
+    //   ),
+    //   borderWidth: number('borderWidth', 1),
+    // }}
```

Comment:
Commented code from sommelier-12 (actually my branch)))). Now 12th fixed, merge with development will fix that (or just erase here)